site/src/components/popover.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Popover
+description: The Popover component in the Atlas Design System
+template: standard
+---
+
+# Popover
+
+The popover component allows content to be placed in the popup that appears after clicking on a button. Popover is based on top of the [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#customizing_the_disclosure_widget) HTML component.
+
+```html
+<details class="popover">
+	<summary class="popover-summary">Click for details</summary>
+	<div class="popover-content">
+		<p class="margin-bottom-xs">It can be a paragraph of text.</p>
+		<p class="margin-bottom-xxs">Or a list:</p>
+		<ul>
+			<li>
+				<a href="#">List item</a>
+			</li>
+			<li>
+				<a href="#">List item</a>
+			</li>
+			<li>
+				<a href="#">List item</a>
+			</li>
+		</ul>
+		<p class="margin-top-xs">Or basically anything else you can think of.</p>
+	</div>
+</details>
+```
+
+You can also use the `.button` class in combination with `.popover-summary` to get the button look.
```

Comment:
## Styling the trigger

You can apply almost any classes in combination with the `.popover-summary` element to achieve the look you need. The following example applies `.button`.

Reply:
Updated